bricks/dart_frog_dev_server/hooks/src/report_external_path_dependencies.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:io' as io;
+
+import 'package:mason/mason.dart';
+import 'package:path/path.dart' as path;
+import 'package:pubspec_parse/pubspec_parse.dart';
+
+Future<void> reportExternalPathDependencies(
+  HookContext context,
+  io.Directory directory,
+) async {
+  final pubspec = Pubspec.parse(
+    await io.File(path.join(directory.path, 'pubspec.yaml')).readAsString(),
+  );
+
+  final dependencies = pubspec.dependencies;
+  final devDependencies = pubspec.devDependencies;
+  final pathDependencies = [...dependencies.entries, ...devDependencies.entries]
+      .where((entry) => entry.value is PathDependency)
```

Comment:
WhereType won't work here?

Reply:
Good catch! I just moved the existing code over but I can adjust this shortly.

Reply:
Actually that won't work because it's not entry, it's entry.value which is casted so going to leave it as is.